examples/k3s/main.tf

```diff
@@ -341,10 +341,13 @@ resource "aws_autoscaling_group" "autoscaling_group" {
   health_check_type  = "ELB"
   capacity_rebalance = false
 
+  # Temporariliy suspending the healthcheck process
+  suspended_processes = ["HealthCheck"]
+
   # Make this really large so we don't check soon :)
-  health_check_grace_period = 10000
-  desired_capacity          = local.desired_size
-  target_group_arns         = [aws_lb_target_group.target_group.arn]
+  # health_check_grace_period = 10000
```

Comment:
Can you explain the need to comment this out? I haven't looked at health checks closely and I don't know how they work, beyond a high level.

Reply:
So, the health checks grace period are for 10000 seconds which is approximately 2hrs and some minutes. after that time, aws ags for some reason kills one instance and respawn another one - at that point flux does not work. 

So, i paused the healthcheck subsystem entirely for the time-being.

Reply:
Ah this makes sense (and it's triggering my memory!) I think I set it to be that hugely high number anticipating we likely wouldn't reach it for our tests, but I didn't read carefully enough that if we just remove the field they won't be done, period! Would you mind adding a comment there for future us to remember?

Reply:
sure.